corehq/apps/callcenter/utils.py

```diff
@@ -23,7 +24,15 @@
 from django.core.cache import cache
 
 
-class DomainLite(namedtuple('DomainLite', 'name default_timezone cc_case_type use_fixtures')):
+@attr.s
+class DomainLite(object):
+    name = attr.ib()
+    default_timezone = attr.ib()
+    cc_case_type = attr.ib()
+    use_fixtures = attr.ib()
```

Comment:
nit: Would it also make sense to set defaults (`None, None, True`) for the above three fields? That would allow the `test_call_center_data_source_provider` generated test cases to omit those arguments.